src/paulscode/android/mupen64plusae/PlayMenuActivity.java

```diff
@@ -71,11 +71,12 @@
     private static final String CATEGORY_GAME_SETTINGS = "categoryGameSettings";
     private static final String CATEGORY_CHEATS = "categoryCheats";
     
-    private static final String ACTION_RESUME = "actionResume";
-    private static final String ACTION_RESTART = "actionRestart";
+    public static final String ACTION_RESUME = "actionResume";
+    public static final String ACTION_RESTART = "actionRestart";
     private static final String ACTION_CHEAT_EDITOR = "actionCheatEditor";
     private static final String ACTION_WIKI = "actionWiki";
     private static final String ACTION_RESET_GAME_PREFS = "actionResetGamePrefs";
+    private static final String ACTION_EXIT = "actionExit";
```

Comment:
After you finish playing and finish() is called in GameActivity, PlayMenuActivity is still on the activity stack, but we want to return directly to GalleryActivity. This new action is used to exit PlayMenuActivity automatically.